utils/marian_client.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+"""
+A client that connects to a Marian server and translates text interactively.
```

Comment:
Great feature.
I am wondering if it would be hard to implement it without a local server?

Reply:
Do you mean as a standalone app? It's not hard, but it must be in C++ since Marian doesn't have Python bindings.